ai/amplify.py

```diff
@@ -25,8 +27,11 @@ async def amplify(self, context, message: str, target_channel: discord.TextChann
         channel_webhook = await webhook.get_webhook_for_channel(target_channel)
 
         for drone in member_drones:
-            await webhook.proxy_message_by_webhook(message_content=f"{get_id(drone.display_name)} :: {message}",
+
+            formatted_message = generate_battery_message(f"{get_id(drone.display_name)} :: {message}")
```

Comment:
3287 :: Doesn't match with what the updated `generate_battery_message()` function does, most likely is broken and needs changing.